### PR TITLE
Add Cohere Fine-tuning docs

### PR DESCRIPTION
**Pull request by morganmcg1**
Adds docs for Cohere fine-tuning capability launching tomorrow.

Tested with `yarn start`, screenshots:

Screenshot 2024-10-02 at 19 40 42

Screenshot 2024-10-02 at 19 40 52

### CODE REVIEW

File: docs/guides/integrations/other/cohere-fine-tuning.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+---
+slug: /guides/integrations/cohere-fine-tunining
+description: How to Fine-Tune Cohere models using W&B.
+displayed_sidebar: default
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cohere Fine-Tuning
```
[the remark this answers is not in the document]
Comment (morganmcg1) [reply docs/guides/integrations/other/cohere-fine-tuning.md]: addressed

File: docs/guides/integrations/other/cohere-fine-tuning.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+With Weights & Biases you can log your Cohere model's fine-tuning metrics and configuration to analyse and understand the performance of your newly fine-tuned models and share the results with your colleagues. 
+
+## Log your Cohere Fine-Tuning Results
+
+To add logging to your W&B workspace:
+
+1. Create a `WandbConfig` with your W&B api key, W&B `entity` and `project` name. You can find your W&B api key at https://wandb.ai/authorize
```
[the remark this answers is not in the document]
Comment (morganmcg1) [reply docs/guides/integrations/other/cohere-fine-tuning.md]: addressed

File: docs/guides/integrations/other/cohere-fine-tuning.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      base_model=...
+      dataset_id=...
+      hyperparameters=...
+      wandb=wandb_ft_config  # pass your W&B config here
+    ),
+  ),
+)
+```
+
+You will then be able to view your model's training metrics and hyperparameters in the W&B project that you created.
```
[the remark this answers is not in the document]
Comment (morganmcg1) [reply docs/guides/integrations/other/cohere-fine-tuning.md]: addressed

File: docs/guides/integrations/other/cohere-fine-tuning.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cohere Fine-Tuning
+
+With Weights & Biases you can log your Cohere model's fine-tuning metrics and configuration to analyse and understand the performance of your newly fine-tuned models and share the results with your colleagues. 
+
+## Log your Cohere Fine-Tuning Results
```
[the remark this answers is not in the document]
Comment (morganmcg1) [reply docs/guides/integrations/other/cohere-fine-tuning.md]: addressed

File: docs/guides/integrations/other/cohere-fine-tuning.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cohere fine-tuning
+
+With Weights & Biases you can log your Cohere model's fine-tuning metrics and configuration to analyse and understand the performance of your newly fine-tuned models and share the results with your colleagues. 
+
+This [guide from Cohere](https://docs.cohere.com/page/convfinqa-finetuning-wandb) has a full example of how to kick off a fine-tuning run and you can find the [Cohere api docs here](https://docs.cohere.com/reference/createfinetunedmodel#request.body.settings.wandb)
```
Comment (johndmulhausen) [inline docs/guides/integrations/other/cohere-fine-tuning.md]:
```suggestion
This [guide from Cohere](https://docs.cohere.com/page/convfinqa-finetuning-wandb) has a full example of how to kick off a fine-tuning run and you can find the [Cohere API docs here](https://docs.cohere.com/reference/createfinetunedmodel#request.body.settings.wandb)
```

File: docs/guides/integrations/other/cohere-fine-tuning.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+description: How to Fine-Tune Cohere models using W&B.
+displayed_sidebar: default
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Cohere fine-tuning
+
+With Weights & Biases you can log your Cohere model's fine-tuning metrics and configuration to analyse and understand the performance of your newly fine-tuned models and share the results with your colleagues.
```
Comment (johndmulhausen) [inline docs/guides/integrations/other/cohere-fine-tuning.md]:
```suggestion
With Weights & Biases you can log your Cohere model's fine-tuning metrics and configuration to analyze and understand the performance of your models and share the results with your colleagues. 
```

File: docs/guides/integrations/other/cohere-fine-tuning.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+# Cohere fine-tuning
+
+With Weights & Biases you can log your Cohere model's fine-tuning metrics and configuration to analyze and understand the performance of your models and share the results with your colleagues. 
+
+This [guide from Cohere](https://docs.cohere.com/page/convfinqa-finetuning-wandb) has a full example of how to kick off a fine-tuning run and you can find the [Cohere API docs here](https://docs.cohere.com/reference/createfinetunedmodel#request.body.settings.wandb)
+
+## Log your Cohere fine-tuning results
+
+To quickly add Cohere fine-tuning logging to your W&B workspace:
```
Comment (johndmulhausen) [inline docs/guides/integrations/other/cohere-fine-tuning.md]:
```suggestion
To add Cohere fine-tuning logging to your W&B workspace:
```

**Review by johndmulhausen — APPROVED** [review]
Fixed a few minor things; approving & merging

**Pull request merged**